Sift/Schema/ComplexTypes/wire.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Wire",
+  "type": "object",
+  "required": [ "$wire_type", "$account_holder_name"],
```

Comment:
@rsomavarapu-sift 

There's an error in the generate step if I add
```
  "oneOf": [
    {
      "required": [
        "$routing_number"
      ]
    },
    {
      "required": [
        "$swift_id"
      ]
    },
    {
      "required": [
        "$bic"
      ]
    }
  ],
```
oneOf is present in code repo (java) and passes using [validator](https://www.jsonschemavalidator.net/). dotnet is having an issue understanding this. 

Including oneOf causes ./build.sh to fail at the generate step where schemas are generated

Reply:
what's the error? We do the validation before ingestion in our API service, so this is not mandatory but good to see if we can do the validation here.

Reply:
Okay. There is no error, but I was not able to add the same validation in yaml

Reply:
Yes, the 3rd party framework that we use doesn't support that: https://github.com/JamesNK/Newtonsoft.Json/issues/964.